package.json

```diff
@@ -77,6 +77,7 @@
     "react-slick": "^0.25.2",
     "react-use": "^14.0.0",
     "rehype-react": "^5.0.1",
+    "remark-preset-lint-recommended": "^5.0.0",
```

Comment:
What was this for again? 🙂

Reply:
It's from @rogermparent 's work [here](https://github.com/iterative/dvc.org/commit/fa56570bb66ad3838e9075569abe3963cfa328a3#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) 

Actually this is my first experience with gatsby or React, so I'm just following his footsteps 😄

Reply:
Ah yes np. I remember what this is about: the tooltips will come from the frontmatter instead of the md content.